### ISSUE TITLE
[13.0] Export All: Extra column "Activity Exception Decoration" on export with Activity view

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#37850]
Impacted versions: 13.0

Steps to reproduce: Export Records with Action/Model where Activities are defined 
Eg, Sales or Purchase 

Current behavior: xls has extra columns called "Activity Exception Decoration"
![Screenshot from 2019-10-03 08-30-45](https://user-images.githubusercontent.com/16624719/66096286-204a6380-e5b8-11e9-9de4-8494c7427253.png)

Expected behavior: No extra column

Video/Screenshot link (optional):

**Comment by msh-odoo** [comment]
@sswapnesh  We already have task for this and WIP branch is here: https://github.com/odoo/odoo/compare/saas-12.5...odoo-dev:saas-12.5-export-activity-exception-ppr

**Comment by ppr-odoo** [comment]
This issue has been fixed at saas-12.5:https://github.com/odoo/odoo/pull/37952

**Comment by sswapnesh** [comment]
@ppr-odoo  Thanks for the Update

I think we have to handle the same here as well

![](https://image.prntscr.com/image/9adgqzaOQX6nyEHKO8V24g.png)

**Comment by burhan-vakharia** [comment]
I confirm this issue on Odoo v13 Community and Enterprise Version.

**Comment by Yenthe666** [comment]
OPW `2256840` since it still exists.

**Comment by msh-odoo** [comment]
@sswapnesh @Yenthe666 @burhan-vakharia The PR #37952 is still not merged, once it is merged forward porting should fix it in 13.0 as well

**Comment by Yenthe666** [comment]
Yeah I made the OPW so somebody from Odoo can take the time to handle that :)

**Comment by Yenthe666** [comment]
Here's the final conclusion from Odoo:

> That is because, from the 'listview' perspective, the "Activity Exception Decoration field" is visible in the treeview.
It is the 'activity_exception' widget with which the field is defined in the view that determines whether the icon is showing or not.
So the export tool does what it was told to do: to preselect fields visible in the listview.
There is currently no way to 'blacklist' field, and we are not keen to do it.
We are aware on the issue but, after discussion with the team leaders of the relevant RD teams, we couldn't find a solution for this yet.

So in short: there's a feedback but it's not going to be changed/fixed anytime soon
